admin/query/src/main/java/org/codice/ditto/replication/admin/query/ReplicationUtils.java

```diff
@@ -101,21 +102,22 @@ public boolean siteIdExists(String id) {
     return siteManager.exists(id);
   }
 
-  public ReplicationSiteField updateSite(
-      String id,
-      String name,
-      AddressField address,
-      @Nullable String rootContext,
-      boolean isDisabledLocal) {
+  public boolean updateSite(
+      String id, String name, AddressField address, @Nullable String rootContext) {
     String urlString = addressFieldToUrlString(address, rootContext);
     ReplicationSite site = siteManager.get(id);
 
     setIfPresent(site::setName, name);
     setIfPresent(site::setUrl, urlString);
-    site.setIsDisabledLocal(isDisabledLocal);
-    siteManager.save(site);
 
-    return getSiteFieldForSite(siteManager.get(site.getId()));
+    try {
+      siteManager.save(site);
+    } catch (ReplicationPersistenceException e) {
+      LOGGER.debug("Unable to saved updated site {} with id {}", site, id, e);
```

Comment:
❓ Should we not log at error or warning level the issue and at debug the stack trace?

Reply:
I do not think so because at warn, the admin cannot do anything to fix it. And at error it's not really a critical error that stops the system.

Reply:
That is why I was suggesting warning and it is also why there are levels such as critical and error. Warning which typically indicates that something happened that won't prevent the system from continuing to function. Without at least logging it at warning, how would an admin know something happened? We shouldn't need to turn on debug to see stuff happened like this. We shou;d turn debug to get more info about stuff that happened.